contrib/scripts/localnet.sh

```diff
@@ -222,20 +261,33 @@ add_genesis_perp_markets_with_coingecko_prices() {
   rm -f $temp_json_fname
 }
 
+add_genesis_perp_markets_offline() {
+  local reserve_amt=$(add_genesis_reserve_amt)
+  price_btc="20000"
+  price_eth="2000"
+  nibid genesis add-genesis-perp-market --pair=ubtc:unusd --sqrt-depth=$reserve_amt --price-multiplier=$price_btc
+  nibid genesis add-genesis-perp-market --pair=ueth:unusd --sqrt-depth=$reserve_amt --price-multiplier=$price_eth
+}
+
 echo_info "Configuring genesis params"
+
 if add_genesis_perp_markets_with_coingecko_prices; then
   echo_success "set perp markets with coingecko prices"
+elif add_genesis_perp_markets_offline; then
+  echo_success "set perp markets with offline defaults"
 else
   echo_error "failed to set genesis perp markets"
   exit 1
 fi
 
 # set validator as sudoer
-$BINARY genesis add-sudo-root-account "nibi1zaavvzxez0elundtn32qnk9lkm8kmcsz44g7xl"
+add_genesis_param '.app_state.sudo.sudoers.root = "'"$val_address"'"'
 
-add_genesis_param '.app_state.oracle.params.twap_lookback_window = "900s"'
-add_genesis_param '.app_state.oracle.params.vote_period = "10"'
-add_genesis_param '.app_state.oracle.params.min_voters = "1"'
+# hack for localnet since we don't have a pricefeeder yet
+add_genesis_param '.app_state.oracle.exchange_rates[0].pair = "ubtc:unusd"'
+add_genesis_param '.app_state.oracle.exchange_rates[0].exchange_rate = "'"$price_btc"'"'
+add_genesis_param '.app_state.oracle.exchange_rates[1].pair = "ueth:unusd"'
+add_genesis_param '.app_state.oracle.exchange_rates[1].exchange_rate = "'"$price_eth"'"'
```

Comment:
The hacky exchange rate values should match the initial mark prices set with `add_genesis_perp_markets_*`